packages/remark-lint-first-heading-level/readme.md

```diff
@@ -135,14 +135,10 @@ a value of `2` can be defined here.
 
 ##### `ok.md`
 
-When configured with `2`.
```

Comment:
again, I'm a bit confused.
These changes appear to be unrelated to the settings, and it's unclear what these changes fix/improve.